sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1558,6 +1566,25 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .checkValue(v => v >= 0 && v <= 1, "The ratio value must be in [0, 1].")
     .createWithDefault(1.0)
 
+  val AGG_OUTPUT_SIZE_RATIO = conf("spark.rapids.sql.agg.outputSizeRatioToBatchSize")
+    .doc("The ratio of the output size of an aggregation to the batch size. ")
```

Comment:
How is this used and how is it different from spark.rapids.sql.agg.skipAggPassReductionRatio? It is not explained well at all here.

Reply:
This piece of code is irrelavant to repartition based agg, so it will be removed by a subsequent commit